templates/broadcom_icos_show_isdp_neighbors.textfsm

```diff
@@ -0,0 +1,18 @@
+Value ID (\S+)
+Value INTERFACE (\S+)
+Value HOLDTIME (\S+)
+Value CAPABILITY ([\w]{1}(?:\s[\w]){0,2})
+Value PLATFORM ((\S+\s\S+)|(\S+))
+Value PORT (.+)
+
+Start
+  # Captures show isdp neighbors for:
+  # Accton AS4610-54P, Accton AS5610-52X, Quanta LY2R, Quanta LB9, DNI AG3448P-R   
+  # Raw data is the same in the case of all those devices
+  ^Capability\sCodes:
+  ^\s+\S+

```

Comment:
```suggestion
  ^\s+\S+\s+-
```

This narrows the scope to a match stricter to the possible capability codes.

Reply:
Correction made